atoma-proxy/src/server/mod.rs

```diff
@@ -250,18 +336,40 @@ async fn get_selected_node(
             error!("No tasks found for model {}", model);
             return Err(StatusCode::NOT_FOUND);
         }
-        Ok(sui
+        // TODO: What should be the default values for the stack entry/price?
+        if total_tokens > STACK_ENTRY_COMPUTE_UNITS {
```

Comment:
Why do we have this check ? Total tokens can be as large as possible, if a node can process it.